shopify/resources/price_rule.py

```diff
@@ -0,0 +1,10 @@
+from ..base import ShopifyResource
+from .discount_code import DiscountCode
+
+class PriceRule(ShopifyResource):
+    def add_discount(self, discount_code = DiscountCode()):
```

Comment:
I'd probably be more explicit here and use restful verbs

`create_discount_code`